client/js/components/CardForm/styles.js

```diff
@@ -0,0 +1,39 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+
+const styles = StyleSheet.create({
+  input: {
+    marginBottom: THEME.spacing.default(1.5),
+  },
+  flexRow: {
+    flexDirection: 'row',
+  },
+  flexField: {
+    marginRight: THEME.spacing.default(0.5),
+    minWidth: 80,
+  },
+  cols2: {
+    flex: 1,
+  },
+  firstChild: {
+    marginRight: 8,
+  },
+  whatIs: {
+    flex: 1,
+    alignItems: 'center',
+    marginTop: 26,
+    marginBottom: 24,
```

Comment:
use flexbox properties, not huge margins.

Reply:
It's used inside of the credit card information which is just for the presentation.
Not a big function or meaning in this component.

Reply:
All of CardForm is `mock` components which will only show to the client